ground/src/main/java/com/google/android/ground/persistence/remote/RemoteDataStore.kt

```diff
@@ -44,7 +44,7 @@ interface RemoteDataStore {
   suspend fun loadTermsOfService(): TermsOfService?
 
   /** Returns all LOIs in the specified survey. Main-safe. */
```

Comment:
Let's also update the method doc while we're at it.

Reply:
Done!